piccolo_admin/endpoints.py

```diff
@@ -560,12 +592,42 @@ def get_meta(self) -> MetaResponseModel:
 
     ###########################################################################
 
-    def get_table_list(self, request: Request) -> t.List[str]:
+    def get_table_list(self) -> t.List[str]:
         """
         Returns a list of all tables registered with the admin.
         """
         return [i.table_class._meta.tablename for i in self.table_configs]
 
+    ###########################################################################
+
+    def get_translation_list(self) -> TranslationListResponse:
+        """
+        Return a list of language codes and names for each available
+        translation.
+        """
+        return TranslationListResponse(
+            translations=[
+                TranslationListItem(
+                    language_code=translation.language_code,
+                    language_name=translation.language_name,
+                )
+                for translation in TRANSLATIONS
```

Comment:
This should be `for translation in self.translations_map.values()` because if we set only English and Croatian in `create_admin(translations=[ENGLISH, CROATIAN])` all translations are in language dropdown and that is not right because languages ​​that are not set in `create_admin` do not work and return `404 Page not found` error.

Reply:
Good catch!